plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergFileWriterFactory.java

```diff
@@ -111,14 +122,15 @@ public IcebergFileWriter createFileWriter(
             ConnectorSession session,
             HdfsContext hdfsContext,
             IcebergFileFormat fileFormat,
-            MetricsConfig metricsConfig)
+            MetricsConfig metricsConfig,
+            Map<String, String> storageProperties)
```

Comment:
Can we have it as `Properties` instead of `Map<String, String>` ?

Reply:
What's the benefit of Properties over Map?

Reply:
I have also realized that the main problem is that the iceberg attribute uses map < string, string >, which is the biggest difference from hive. If you want to change to properties, it will change a lot. What's your opinion?

Reply:
I was thinking on re-using the code from `HiveUtil` for `withBloomFilterOptions` - so that we could have all the orc related logic at one end. WDYT @alexjo2144 ?